pandas/io/excel/_base.py

```diff
@@ -536,7 +536,7 @@ class ExcelWriter(metaclass=abc.ABCMeta):
     datetime_format : string, default None
         Format string for datetime objects written into Excel files
         (e.g. 'YYYY-MM-DD HH:MM:SS')
-    mode : {'w' or 'a'}, default 'w'
+    mode : 'w' or 'a', default 'w'
```

Comment:
This was intentional per the docstring guide so will want to revert:

https://pandas.pydata.org/pandas-docs/stable/development/contributing_docstring.html#parameter-types

Reply:
@WillAyd Thanks, I didn't see that! As per the same, should it be `mode : {'w', 'a'}`? (comma-separated, default first)

Reply:
Yea nice catch - that makes sense to do here